packages/phone-number-privacy/combiner/test/end-to-end/tmpBackwardsCompatibility.test.ts

```diff
@@ -1,52 +1,58 @@
-import { OdisUtils } from '@celo/identity'
+import { newKit } from '@celo/contractkit-prev'
+import { OdisUtils } from '@celo/identity-prev'
+import { SignMessageRequest } from '@celo/identity-prev/lib/odis/query'
 import { ErrorMessages } from '@celo/identity/lib/odis/query'
-import {
-  AuthenticationMethod,
-  Endpoint,
-  SignMessageRequest,
-} from '@celo/phone-number-privacy-common'
+import { AuthenticationMethod, Endpoint } from '@celo/phone-number-privacy-common'
+import { replenishQuota } from '@celo/phone-number-privacy-common/lib/test/utils'
 import { genSessionID } from '@celo/phone-number-privacy-common/lib/utils/logger'
+import { ensureLeading0x } from '@celo/utils/lib/address'
 import 'isomorphic-fetch'
-import { replenishQuota } from '../../../common/src/test/utils'
-import { VERSION } from '../../src/config'
 import {
   ACCOUNT_ADDRESS,
   ACCOUNT_ADDRESS_NO_QUOTA,
   BLINDED_PHONE_NUMBER,
-  contractKit,
+  DEFAULT_FORNO_URL,
   dekAuthSigner,
+  deks,
   PHONE_NUMBER,
+  PRIVATE_KEY,
+  PRIVATE_KEY_NO_QUOTA,
   SERVICE_CONTEXT,
-  walletAuthSigner,
 } from './resources'
 
 require('dotenv').config()
 
-export const SIGN_MESSAGE_ENDPOINT = '/getBlindedMessageSig'
-
 jest.setTimeout(60000)
 
-describe('Running against a deployed service', () => {
-  beforeAll(() => {
-    console.log('ODIS_COMBINER_SERVICE_URL: ' + process.env.ODIS_COMBINER_SERVICE_URL)
-    console.log('ODIS_BLOCKCHAIN_PROVIDER: ' + process.env.ODIS_BLOCKCHAIN_PROVIDER)
-  })
+const contractKit = newKit(DEFAULT_FORNO_URL)
+contractKit.addAccount(PRIVATE_KEY_NO_QUOTA)
+contractKit.addAccount(PRIVATE_KEY)
+contractKit.defaultAccount = ACCOUNT_ADDRESS
 
-  // This test is disabled because the Combiner status endpoint doesn't work
-  xit('Service is deployed at correct version', async () => {
-    const response = await fetch(process.env.ODIS_COMBINER_SERVICE_URL + Endpoint.STATUS, {
-      method: 'GET',
-    })
-    const body = await response.json()
-    // This checks against local package.json version, change if necessary
-    expect(body.version).toBe(VERSION)
+const combinerUrl = process.env.ODIS_COMBINER_SERVICE_URL
```

Comment:
it may make sense to hard-code in the old combiner URL here to make sure that we are testing against the old one (or use a separate env var but that feels like overkill)